ParallelWithTestNG/src/main/java/WebTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Created by Nghi Thai on 12/29/17.
+ */
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import org.testng.ITestContext;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import java.util.Map;
+
+public class WebTest {
+    public static Map<String, String> testngParams;
+    String kobitonURL;
+    RemoteWebDriver driver = null;
+
+    @BeforeTest
+    @Parameters({"platform"})
+    public void Setup(ITestContext testContext, String platform) throws MalformedURLException {
+        testngParams = testContext.getCurrentXmlTest().getAllParameters();
+        kobitonURL = testngParams.get("kobitonURL");
+        driver = new RemoteWebDriver(new URL(kobitonURL), this.generateDesiredCaps(platform));
+        driver.manage().timeouts().implicitlyWait(90, TimeUnit.SECONDS);
+    }
+
+    @AfterTest
+    public void Teardown() {
+        try {
+            if (driver != null)
+                driver.quit();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public void testKobitonWebsite() {
+        driver.get("http://google.com");
+        WebElement searchFieldE = driver.findElementByXPath("//input[@name='q']");
+        searchFieldE.sendKeys("Kobiton");
+        searchFieldE.sendKeys(Keys.ENTER);
+        String message = driver.getTitle();
+        Assert.assertTrue(message.contains("Kobiton"));
+    }
+
+    public static DesiredCapabilities generateDesiredCaps(String platform) {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        String browserName = "chrome";
+        String deviceName = "Galaxy";
```

Comment:
Put the deviceName as param in `testng.xml`

Reply:
Put more parameters: `deviceName`, `browserName` in testng.xml file